hack/actions/install-kubernetes-toolchain.sh

```diff
@@ -0,0 +1,66 @@
+#! /usr/bin/env bash
```

Comment:
can these be incorporated into https://github.com/envoyproxy/gateway/blob/main/tools/make/kube.mk which is doing something similar (installing kube tools)